toqito/nonlocal_games/xor_game.py

```diff
@@ -290,3 +290,20 @@ def classical_value(self) -> float:
             "multiple repetitions for the classical value of "
             "a nonlocal game."
         )
+    def to_nonlocal_game(self) -> np.ndarray:
```

Comment:
I wonder if we want to refactor this to return a `NonlocalGame` object. 

So:

```python
        q_0, q_1 = self.prob_mat.shape
        pred_mat = self.pred_mat
        xor_pred_mat = np.ndarray((2, 2, q_0, q_1))

        for a in range(2):
            for b in range(2):
                for x in range(q_0):
                    for y in range(q_1):
                        xor_pred_mat[a, b, x, y] = pred_mat[x, y] == a ^ b

        return NonlocalGame(self.prob_mat, xor_pred_mat, reps=self.reps)
```

*Note*: Haven't tested the above, but something like this